studio/components/interfaces/Reports/Reports.constants.ts

```diff
@@ -183,7 +183,44 @@ export const PRESET_CONFIG: Record<Presets, PresetConfig> = {
           avg desc
         limit
         3
-        `
+        `,
+      },
+      networkTraffic: {
+        queryType: 'logs',
+        sql: (filters) => `
+        select
+          cast(timestamp_trunc(t.timestamp, hour) as datetime) as timestamp,
+          coalesce(
+            safe_divide(
+              sum(
+                cast(coalesce(headers.content_length, "0") as int64)
+              ),
+              1000000
+            ),
+            0
+          ) as ingress_mb,
+          coalesce(
+            safe_divide(
+              sum(
+                cast(coalesce(resp_headers.content_length, "0") as int64)
```

Comment:
Is this even set for more requests? Remember this was missing for a lot of requests if not all

Reply:
would not be included for certain responses if the response is empty. However, for significant requests such as GETs to storage or POSTs to storage, it would be present.

it is a temporary solution until we get the data warehouse connection to LF.